livekit-android-sdk/src/main/java/io/livekit/android/room/participant/LocalParticipant.kt

```diff
@@ -1232,3 +1608,23 @@ internal fun VideoTrackPublishOptions.hasBackupCodec(): Boolean {
 
 private val backupCodecs = listOf(VideoCodec.VP8.codecName, VideoCodec.H264.codecName)
 private fun isBackupCodec(codecName: String) = backupCodecs.contains(codecName)
+
+/**
+ * A handler that processes an RPC request and returns a string
+ * that will be sent back to the requester.
+ *
+ * Throwing an [RpcError] will send the error back to the requester.
+ *
+ * @see [LocalParticipant.registerRpcMethod]
+ */
+typealias RpcHandler = suspend (requestId: String, callerIdentity: Participant.Identity, payload: String, responseTimeoutMs: Int) -> String
```

Comment:
the arguments should be provided as an `RpcInvocationData` data class so that we can add additional arguments in the future without breaking existing handlers